packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -477,6 +477,7 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
         },
         hoverAction: () => {
           this._onLegendHover(singleChartData.legend);
```

Comment:
Should handleChartMouseLeave come first.

Reply:
I think this doesn't make much of a difference result-wise as `setState` is asynchronous. But it will surely be easier to understand if `handleChartMouseLeave` comes first